.env

```diff
@@ -1,2 +1,2 @@
-VUE_APP_API_URL=https://api.qmk.fm
-VUE_APP_KEYBOARDS_URL=https://keyboards.qmk.fm
+VITE_API_URL=https://api.qmk.fm
+VITE_KEYBOARDS_URL=https://keyboards.qmk.fm
```

Comment:
```suggestion
VITE_APP_KEYBOARDS_URL=https://keyboards.qmk.fm
```

Reply:
I think documentation says it's prefix is https://vitejs.dev/config/#envprefix am I reading it wrong?

Reply:
you're right i was confused by this one: https://vitejs.dev/guide/env-and-mode.html#modes

Reply:
I know the doc is really confusing. I think the namespace would be APP_* which in itself isn't a terrible idea.